modules/meb_api/app/controllers/meb_api/v0/education_benefits_controller.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 
+require 'dgi/eligibility/service'
```

Comment:
Does the meb_api module have API docs as of yet? I'm not seeing any currently

Reply:
No we don't have API/Swagger docs as of yet but are aware they will be needed as the API gets finalized. These routes are behind a feature flag that is disabled in production. The goal for this is to make sure that the two environments can communicate in sandbox/staging environment.